### ISSUE TITLE
Header: replace Healthlocker logo with the Menu item area title (not homepage)

### ISSUE THREAD

**Issue by MollieCourtenay** [healthlocker/healthlocker#466]
Examples:

<img width="533" alt="screen shot 2017-04-21 at 11 47 34" src="https://cloud.githubusercontent.com/assets/25012688/25274545/656e6106-2688-11e7-80af-b2d315585729.png">
<img width="534" alt="screen shot 2017-04-21 at 11 46 06" src="https://cloud.githubusercontent.com/assets/25012688/25274546/65718c3c-2688-11e7-8f3e-525ef6a9cb36.png">

**Comment by iteles** [comment]
Agreed with @katyedwards that this would be deprioritised out of sprint 11 for now.

**Comment by katyedwards** [comment]
@MollieCourtenay for the length of time this would take to implement, it didn't seem like the best use of time for this week's work, but can always be added in later if more sprints are added.
